docs/product/reference/search/searchable-properties/session-replay.mdx

```diff
@@ -48,7 +48,7 @@ The `class` of an element that was clicked. No leading `.` is necessary. For exa
 
 The name of the frontend component that was clicked. For example, `MyAwesomeComponent` would match the React component `<MyAwesomeComponent>`.
 
-Note: This property requires that your project have [React component name tracking configured](/platforms/javascript/guides/react/features/component-tracking/#component-name-tracking).
```

Comment:
do we have redirects in place?

Reply:
what do you mean by redirects? How do you set that up?

Reply:
if this is only under `guides/react` you don't need to worry about the regex group to catch all, like here: https://github.com/getsentry/sentry-docs/blob/5467a3ef3ccd4daa9b63f39051525ddbf86aba84/vercel.json#L546-L552

You can just have `/guides/react/track-..` etc

Reply:
I added the redirect!